base/src/test/resources/success/common/src/Arith/Nat/Equality.aya

```diff
@@ -0,0 +1,20 @@
+open import Paths
+open import Arith::Nat::Core
+open import Arith::Nat::Properties
+open import Logic::Dec
+open import Logic::False
+
+def ≐-suc {m n : Nat} (p : Dec (m = n)) : Dec (suc m = suc n)
+| yes p => yes (pmap suc p)
+| no np => no (\ p =>
+  // p  : suc m = suc n
+  // np : m = n -> False
+  // Goal : False
```

Comment:
Are these information available in vscode?

Reply:
No, I list them manually for me.

Reply:
I think they are

Reply:
Use a hole, `{??}` as an expression